notes/points-markets-settlement.md

```diff
@@ -0,0 +1,103 @@
+
+*mean conversion rate*
+: The average rate in terms of the `Base Asset` at which each point involved in an *Airdrop Event* underwent *conversion*.
+
+*termination date/time*
+: The time at which trading on the market should stop as a result of an ***Imminent Airdrop Event***.
+
+
+### Variables
+
+The following terms (with suggested values in brackets) used in the definitions refer to quantities or other parameters that affect the specifics of the product. They may be varied to suit each individual use case.
+
+`Base Asset` (USDT)
+: The asset in which *points* that have undergone `conversion` are valued.
+
+`Inclusivity Threshold` (40.00%)
+: The percentage of issued points that must be eligible for the conversion
+
+`Liquidity Threshold` (5.00%)
+: The percentage of issued points that must be met by the bid depth (quantity of resting buy orders or points otherwise able to be sold) in the market within `valid slippage` (%) of the current market price.
+
+`Valid Slippage` (20.00%)
+: The range around the current market price at which available bids are deemed to constitute ‘liquidity’ for the purpose of an ***Airdrop Event*** in which points have become tradable.
+
+`Minimum Trading Window` (48 hours)
+: The minimum amount of cumulative time over which a market meeting the liquidity and slippage thresholds must exist to trigger an ***Airdrop Event***.
+
+`Early Termination Period` (24 hours)
+: The amount of time before a scheduled ***Airdrop Announcement***
+
+
+### Airdrop Event
+
+An ***Airdrop Event*** is deemed to have occurred when one or more of the following conditions is satisfied:
+
+1. At least `Inclusivity Threshold` (%) of all issued *points* have undergone *conversion* (defined above) since the inception of the points programme.
+
+2. It is no longer possible for *conversion* to happen, for example if the protocol ceases operation. (In this case the *mean conversion rate* will be zero.)
+
+3. Points become freely tradable by the majority of points holders to the extent that a current market price per point exists and the cumulative historic trading volume plus the available liquidity within `Valid Slippage` of the current market price is at least `Liquidity Threshold` (%) of the total number of issued points for a (not necessarily contiguous) period of no less than `Minimum Trading Window`.  
+
+Note: adjustments to points balances such as normal or reverse splits (1 point becomes, 2 points, 10 points, 100 points, 0.5 points, 0.1 points, etc.), multiplying some users points by some value, etc. etc. do not constitute an *Airdrop Event* and simply impact the market price of a point as an issuance of new points would.
+
+Once an ***Airdrop Event*** has occurred and all input to the ***Valuation Algorithm*** is available, the *mean conversion rate* is calculated and the market will be settled. 
+
+
+### Imminent Airdrop Event
```

Comment:
I am still unsure whether we need this (arguably not as well defined event as the airdrop event). 

In my mind the market could keep trading just fine until the valuation is obtained, at this point trading terminates and (say) a minute later the market settles. 

I don't see what is achieved by terminating trading on "imminent airdrop event" and then keeping the market suspended (potentially over days when prices can shift) until settlement. 

Perhaps having this defined causes no harm and finally people can setup the points markets as they see fit (with trading terminated on imminent airdrop event or trading terminated only when price becomes available).

Reply:
I think, if I'm reading correctly, the "imminent airdrop event" event itself only triggers `Early Termination Period` before the actual market distribution of tokens. i.e. you would have a flow:

1. "Early Termination Period" = 1hr
2. T - 3days: Airdrop announced at time T with conversion rate for points to tokens X
    - This triggers an "Imminent Airdrop Event" vote, for market termination at T - 1hr
3. Vote goes through
4. T - 1hr: Market ends trading
5. T: Airdrop occurs and prices measured

Reply:
That makes sense. Though wouldn't it be "market termination at T + 1hr" in step 2. ?

Reply:
@davidsiska-vega 
Do you mean it to be "market termination at T - 1hr" in step 2.

T minus one as people should know before the termination?

Reply:
> I am still unsure whether we need this (arguably not as well defined event as the airdrop event).
> 
> In my mind the market could keep trading just fine until the valuation is obtained, at this point trading terminates and (say) a minute later the market settles.

Feedback from community LPs is that they would rather the markt settle some time (even if only a short time—it need not be days) before the actual information needed to _know_ the conversion rate is available.

So, similar to a future with a known expiry terminating shortly before the expiry timestamp to ensure nobody could ever trade while in posession of knowledge of the final settlement price. Admittedly this may not be possible if there's no pre-announcement, but the 'Imminent Airdrop Event' is intended to catch cases where it is possible.

Are there any clarifying changes we could make to this section to improve it?

Reply:
> that they would rather the markt settle some time (even if only a short time—it need not be days) before the actual information needed to know the conversion rate is available

With "terminate" instead of settle this makes sense. 

A line including the rationale that the concept "imminent airdrop event" "allows market creators to minimise the chance that someone in posession of the final settlement price can trade" may have been handy for me.